shop-ui/build.gradle

```diff
@@ -30,21 +30,18 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    dataBinding {
```

Comment:
try to enable this only in `core-ui` module

Reply:
tried, but it should be included in all modules where needed